apps/nuxt3-ssr/package.json

```diff
@@ -37,6 +38,10 @@
     "vitest": "^0.30.1"
   },
   "dependencies": {
-    "floating-vue": "^2.0.0-beta.20"
+    "@unhead/vue": "1.1.30",
+    "floating-vue": "^2.0.0-beta.20",
```

Comment:
fix version

Reply:
yep not part of this PR, the line is only moved